base/formats_test.go

```diff
@@ -0,0 +1,33 @@
+package base
+
+import (
+	"bufio"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+)
+
+func TestUtil_GetFileExtension(t *testing.T) {
+	c := qt.New(t)
+
+	file, err := os.Open("./testdata/test_image.png")
+	c.Assert(err, qt.IsNil)
+	defer file.Close()
+	wantFileExtension := "png"
+
+	reader := bufio.NewReader(file)
+	content, err := io.ReadAll(reader)
+	c.Assert(err, qt.IsNil)
+
+	fileBase64 := base64.StdEncoding.EncodeToString(content)
+	fileBase64 = "data:image/png;base64," + fileBase64
+	fmt.Println(fileBase64)
```

Comment:
NIT
Sorry, I missed this. We probably want to take it out.

Reply:
Oh yeah, I missed it too.

Reply:
fixed